2018/07/28 14:05:30 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:31 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:32 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:32 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:32 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:32 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:32 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:05:40 [error] 5621#5621: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:06:48 [error] 5621#5621: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:14:05 [error] 5621#5621: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:14:53 [error] 5621#5621: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:14:54 [error] 5621#5621: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:14:54 [error] 5621#5621: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:14:55 [error] 5621#5621: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:14:55 [error] 5621#5621: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
2018/07/28 14:16:00 [error] 6789#6789: *26 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: umbrellio.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "umbrellio.local"
